tests/unit/s2n_self_talk_ktls_test.c

```diff
@@ -288,6 +291,8 @@ int main(int argc, char **argv)
             EXPECT_BYTEARRAY_EQUAL(file_test_data + offset, buffer, read);
         }
 
+        EXPECT_SUCCESS(close(file));
```

Comment:
nit, but it doesn't seem ideal to have `file` defined in the global test scope but have it unusable after this test due to the close. It also seems pretty specific to s2n_sendfile tests, so it might make sense to move these global definitions into a scope just for s2n_sendfile:
```
/* Test: s2n_sendfile */
{
    uint8_t file_test_data[100] = { 0 };
    int file = open(argv[0], O_RDONLY);
    ...
    for (size_t offset_i = 0; offset_i < s2n_array_len(test_offsets); offset_i++) {
        ...
    }

    EXPECT_SUCCESS(close(file));
}
```